### PR TITLE
Fix Aurelia template to lower the entry level

### PR DESCRIPTION
**Pull request by MaximBalaganskiy**
Added jQuery and made it available globally
Added Promise polyfill to support IE ootb
Fixed vendor bundle to have correct format modules

### CODE REVIEW

**Comment by SteveSandersonMS** [comment]
Thanks for the contributions!

For us to be able to include some of these changes, we'd need more information on what benefits they offer (or possible drawbacks). Ideally, it's best to first open an issue at https://github.com/aspnet/javascriptservices/issues (which is where we design and plan the SPA templates - I know this isn't obvious though!) to discuss the proposals.

As for this PR, we probably wouldn't want to expose `jQuery` to global scope as that's not great practice in modern SPA development. For the IE shims, this is something we'll be addressing in a common way across all the SPA templates, so you don't need to cover that. I'd like to know what `AureliaPlugin` does and why it was necessary to import `ModuleDependenciesPlugin` even though it doesn't seem to be used. And finally, I know this is a minor detail, but the code style doesn't quite match the rest of the file (e.g., use of `"` quote and `const`/`var`).

I'll close this pending your further discussion at https://github.com/aspnet/javascriptservices/issues. Hope that's OK!

**Comment by PeterWone** [comment]
The don't-strew-your-junk-around-the-global-scope rule exists to prevent accidental collisions. Just about everything ever written depends jQuery. Where is it expected to be? **In the global scope, as jQuery and as $.** If you won't let something as ubiquitous as jQuery use it, the global scope might as well not exist.

**Comment by SteveSandersonMS** [comment]
@PeterWone We're not stopping people from putting jQuery in global scope if they want. That's a choice for each developer. It's just not required by default.

**Comment by PeterWone** [comment]
@SteveSandersonMS  I understand your position quite well and for me or Max it's fine. The flaw with this approach is that it only caters to the needs of people who understand the issues we're discussing. I don't believe these are the primary users of your templates. I could be wrong about that; I have no statistics, perhaps you do.

File: src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/webpack.config.vendor.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ var path = require('path');
 var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var extractCSS = new ExtractTextPlugin('vendor.css');
+const { AureliaPlugin, ModuleDependenciesPlugin } = require("aurelia-webpack-plugin");
```
Comment (AdamWillden) [inline src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/webpack.config.vendor.js]: `ModuleDependenciesPlugin` is unused

Comment (MaximBalaganskiy) [reply src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/webpack.config.vendor.js]: yeah, that's a leftover... should be removed. Much of the PR is not getting in, does it make sense to just create a new one?

**Comment by SteveSandersonMS** [comment]
> does it make sense to just create a new one?

@MaximBalaganskiy Yes, if that's OK. Thanks!

**Pull request closed without merging**